packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -44,29 +48,19 @@ describe("LabeledTextField", () => {
         );
     });
 
-    it("value state changes when the user types", () => {
```

Comment:
I agree that this test doesn't make sense anymore, but it would be still good to test that the component is being controlled (e.g. test that when `value` is changed after being mounted, then `onChange` is triggered with the correct new value.)

BTW I'm not sure if you moved this test or just planning to delete it.

Reply:
I indeed test the `onChange` prop being called on input change. I guess GitHub kind of hides the test title, but it is on line 222 on the same `"labeled-text-field.test.js"` file. This is the exact code it has:
```js
it("onChange prop is called on input change", () => {
        // Arrange
        const handleChange = jest.fn((newValue: string) => {});

        const wrapper = mount(
            <LabeledTextField label="Label" value="" onChange={handleChange} />,
        );

        // Act
        const newValue = "new value";
        const input = wrapper.find("input");
        input.simulate("change", {target: {value: newValue}});

        // Assert
        expect(handleChange).toHaveBeenCalledWith(newValue);
});
```